specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -1564,922 +2662,1213 @@
+        "windSpeed": {
+          "$ref": "#/definitions/WeatherUnit",
+          "description": "Wind speed associated with the radiusSectorData."
+        },
+        "radiusSectorData": {
           "type": "array",
-          "description": "Details of the weather hazards affecting the trip.",
+          "description": "Contains the information needed to plot wind radius quadrants. Bearing 0–90 = NE quadrant; 90–180 = SE quadrant; 180–270 = SW quadrant; 270–360 = NW quadrant.",
           "items": {
-            "$ref": "#/definitions/HazardDetail"
+            "$ref": "#/definitions/RadiusSector"
           }
+        },
+        "radiiGeometry": {
+          "$ref": "../../../Common/preview/1.0/common.json#/definitions/GeoJsonGeometry",
+          "description": "GeoJSON object. Displayed when radiiGeometry=true in request. Describes the outline of the wind radius quadrants."
         }
       }
     },
-    "HazardDetail": {
+    "RadiusSector": {
       "type": "object",
+      "description": "Bearing 0–90 = NE quadrant; 90–180 = SE quadrant; 180–270 = SW quadrant; 270–360 = NW quadrant.",
       "properties": {
-        "hazardIndex": {
-          "$ref": "#/definitions/HazardIndex"
+        "beginBearing": {
+          "type": "number",
+          "format": "double",
+          "description": "Bearing, in degrees, of the beginning of the quadrant."
         },
-        "hazardCode": {
-          "type": "string",
-          "description": "A unique identifier (non-displayable) for each type of hazard: LightRain, ModerateRain, HeavyRain, LightMix, ModerateMix, HeavyMix, LightSnow, ModerateSnow, HeavySnow, LightIce, ModerateIce, HeavyIce, Hail, LargeHail, SunGlare, SunGlareHigh, Lightning, SevereLightning, WindModerate, WindHigh, WindExtreme, FloodWarning, FlashFloodWarning, TornadoWarning, TsunamiWarning, SevereThunderstormWarning."
+        "endBearing": {
+          "type": "number",
+          "format": "double",
+          "description": "Bearing, in degrees, of the end of the quadrant."
         },
-        "shortPhrase": {
-          "x-ms-client-name": "shortDescription",
-          "type": "string",
-          "description": "A displayable short phrase describing the forecasted conditions and precipitation intensity/type."
+        "range": {
+          "type": "number",
+          "format": "double",
+          "x-ms-client-name": "radius",
+          "description": "The radius of the quadrant, in nautical miles."
         }
       }
     },
-    "WeatherNotification": {
+    "StormLocationsResult": {
       "type": "object",
+      "description": "Locations for an individual government-issued storm",
       "properties": {
-        "type": {
+        "results": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LocationResult"
+          },
+          "description": "Locations for an individual government-issued storm"
+        }
+      }
+    },
+    "LocationResult": {
```

Comment:
This should be something like `StormData` or `StormProperties`. Storm location is merely one field among many.

Reply:
This api is like a proxy api for AccuWeather's storm positon api. http://apidev.accuweather.com/developers/tropicalAPIGuide#PositionsGov It not only contains the location info, but some other storm properties. I think it's ok to stick to the naming in AccuWeather to avoid confusion. How do you think?

Reply:
That API doesn't specify any kind of name, so I'm not sure how not calling it "LocationResponse" would make it confusing. It's already confusing since it contains a whole lot more than location info. Lots and lots of Maps APIs have locations in them, so this is not a good class name. `StormLocation` would be okay here. Since this is a collection item, it should not include Response in the name.

Reply:
Thanks for the suggestion!
I have updated it to StormLocation.